ui/package.json

```diff
@@ -39,13 +43,18 @@
     "@fontsource/quicksand": "^4.4.5",
     "@fortawesome/fontawesome-free": "^5.9.0",
     "@stomp/ng2-stompjs": "^8.0.0",
+    "axios": "^0.24.0",
+    "classnames": "^2.3.1",
```

Comment:
Is this the library that just chains classnames?

Reply:
I think so. @mschu124 added it.

Reply:
Yes, it's the react way of doing this in angular:
https://github.com/FordLabs/retroquest/blob/d4a6f28b9c1284779409902a81a2432b9f617f8b/ui/src/app/modules/components/button/button.component.ts#L26-L30